floor_generator/lib/misc/type_utils.dart

```diff
@@ -4,7 +4,9 @@ import 'package:analyzer/dart/constant/value.dart';
 import 'package:analyzer/dart/element/element.dart';
 import 'package:analyzer/dart/element/type.dart';
 import 'package:floor_generator/misc/annotations.dart';
+import 'package:floor_generator/misc/constants.dart';
 import 'package:source_gen/source_gen.dart';
+import 'package:sqlparser/sqlparser.dart' show BasicType;
```

Comment:
❤️ You really know got to know Dart in the meanwhile!

Reply:
Well I added a `View` class to both sqlparser and floor and now I have to cope with the consequences :smile: